app/src/main/java/com/postpc/tenq/network/SpotifyApiService.java

```diff
@@ -48,7 +48,13 @@
     @PUT("playlists/{playlist_id}/tracks")
     Call<Void> reorderPlaylist(@Path("playlist_id") String playlistId, @Body() Map<String, Object> details);
 
+    @PUT("playlists/{playlist_id}/followers")
+    Call<Void> followPlaylist(@Path("playlist_id") String playlistId, @Body() Map<String, Object> details);
```

Comment:
I'm thinking about the flow of room for guests.... I think when joining a room, this api should be called, no?
And when clicking "export", a new identical playlist should be created.
If the user just follows, the playlist might change if the host manually changes it in his Spotify account. What do you think?